tools/docker/scripts/build_cp2k_cmake.sh

```diff
@@ -63,11 +70,14 @@ elif [[ "${PROFILE}" == "toolchain" ]] && [[ "${VERSION}" == "ssmp" ]]; then
     -DCP2K_USE_SPGLIB=ON \
     -DCP2K_USE_LIBXSMM=ON \
     -DCP2K_USE_VORI=ON \
+    -DCP2K_USE_TREXIO=ON \
+    -DCP2K_USE_GRPP=ON \
+    -DCP2K_USE_HDF5=ON \
     -DCP2K_USE_MPI=OFF \
     -DCP2K_USE_MPI_F08=OFF \
     -DCP2K_USE_COSMA=OFF \
     -DCP2K_USE_DLAF=OFF \
-    -DCP2K_USE_LIBTORCH=ON \
+    -DCP2K_USE_LIBTORCH=OFF \
```

Comment:
```suggestion
    -DCP2K_USE_LIBTORCH=ON \
```
This should be re-enabled (see comment above).